### PR TITLE
Renamings

### PR DESCRIPTION
**Pull request by vext01**
Rename everything that refers to traceme, and a small update to the README.

### CODE REVIEW

**Comment by vext01** [comment]
Oh, we've confused travis, one moment.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
-# traceme
+# hwtracer
 
-A small Rust/C library to trace an arbitrary thread using Intel Processor Trace
-(PT).
+A small Rust/C library to trace a program using CPU tracing technology.
+
+Initially this library will support Intel Processor Trace, but may one day
```
Comment (ltratt) [inline README.md]: "will support" -> "only supports".

", but may one day" -> ". We hope in the future to support alternatives such as Arm's CoreSight."

**Comment by ltratt** [comment]
OK, just that one minor comment on the README.

**Comment by vext01** [comment]
Fixed, with a small rewording for flow.

**Comment by ltratt** [comment]
Works for me. If you squash, I can merge.

**Comment by vext01** [comment]
Splat.

**Comment by vext01** [comment]
Travis should wake up on our next PR.

**Comment by ltratt** [comment]
Hmm, what happened? I'd like Travis to work on this one too!

**Comment by vext01** [comment]
We renamed the repo, and I forgot to sync the CI account before raising this PR. If we close this and re-open it should work (tm).

**Pull request closed without merging**